src/nupic/proto/Map.capnp

```diff
@@ -0,0 +1,10 @@
+@0xdc5b07d7180bb807;
+
+struct Map(Key, Value) {
```

Comment:
Maybe I should use this in TP serialization. I'm using a field for each dictionary key in _stats.